docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
 
-To gain more access control adjustments in a project, you can set
-:guilabel:`Access control` to :guilabel:`Custom` to switch over to
-using the Django admin-interface instead of the one in Weblate.
+    - General access (for browsing) to the whole project ``foo`` including both
+      its subcomponents: ``foo/bar`` and ``foo/baz``.
+    - Review strings in ``foo/bar`` Spanish translation (and only it).
+    - Manage VCS for the whole ``foo/bar`` repository e.g. commit pending
+      changes made by translators for all languages.
 
-If you want to do this by default for all current and new projects, configure the
-:setting:`DEFAULT_ACCESS_CONTROL` to administrate all permissions and relations using
-the Django admin interface.
+.. _autogroup:
 
-.. warning::
+Automatic group assignments
++++++++++++++++++++++++++++
 
-    By turning this on, Weblate will remove all :ref:`acl` it has created for
-    this project. If you are doing this without admin permission from the instance, you
-    will instantly lose your access to manage the project.
+On the bottom of :guilabel:`Group` editing page in the
+:ref:`Django admin interface <admin-interface>` you can specify
+:guilabel:`Automatic group assignments`, which is a list of regular expressions
+used to automatically assign newly created users to the group based on their
+e-mail addresses. This assignment only happens upon account creation.
 
-.. _default-groups:
+The most common use case for the feature is to assign all new users to some
+default group. In order to do so you will probably want to keep the default
+value (``^.*$``) for the regexp. Another use case for this option might be to
+give some additional privileges to employees of your company by default.
+Assuming all of them use corporative e-mail addresses on your domain, this can
+be accomplished with an expression like ``^.*@mycompony.com``.
+
+.. note::
+
+    Automatic group assignment to `Users` and `Viewers` is always recreated
+    during migrations. If you want to turn it off, set the regular expression to
+    ``^$`` (which won't match anything).
+
+.. note::
+
+    As for now there is no way to bulk-add already existing users to some group
+    via UI. For that you may resolve to the :ref:`REST API <api>`.
 
 Default groups and roles
-------------------------
-These roles and groups are created upon installation. The built-in roles are always
-kept up to date by the database migration when upgrading.
-Custom changes are not lost. Please define a new role if you want to define your
-own set of permissions.
+++++++++++++++++++++++++
+
+After installation a default set of groups is created (see :ref:`default-groups`).
+
+These roles and groups are created upon installation. The built-in roles are
+always kept up to date by the database migration when upgrading. You can't
+actually change them, please define a new role if you want to define your own
+set of permissions.
+
+.. _privileges:
```

Comment:
Is this intentional? The `privileges` label is removed from top of the document and added here. That makes all references point to a different location now.

Reply:
@Fat-Zer

Reply:
@nijel after the reading, I suggest:
- keeping the change
- change the existing links to `access-control

Reply:
Yes, this was intentional, I've actually tried to review all the references to the document, but it looks like I've missed this one.

Thanks for pointing this out. I'll check all the references to the doc once again to be on the safe side...

Reply:
I fixed the references, please verify.

Reply:
Safer approach would to avoid reusing previous label - the build would fail, and it would be clear what needs to be changed.

Reply:
I like the rewrite. Very boolean.

Would it fix anything to clarify default groups must be removed if you want to add new ones?
Automate that, I don't know, things.

Reply:
This is solved.